includes/MirahezeIRCRCFeedFormatter.php

```diff
@@ -51,6 +51,7 @@ public function getLine( array $feed, RecentChange $rc, $actionComment ) {
 		 */
 
 		$trustAndSafety = [
```

Comment:
Maybe generate a config and do this in mw-config? Easier?

Reply:
Yeah I was just thinking that also, I may be able to pull from the user group members also.

Reply:
PR updated.